### ISSUE TITLE
404 for icon fonts on heroku :(

### ISSUE THREAD

**Issue by dkhgh** [DefactoSoftware/proof#65]
![screen shot 2015-10-31 at 15 13 49](https://cloud.githubusercontent.com/assets/4135610/10864073/1013f89a-7fe2-11e5-8d88-1a3b8a765419.png)

**Comment by dkhgh** [comment]
the following code fixed this in 31fff7152eadee25306bf8a94c4423a6eac3dba0

``` scss
@font-face {
  font-family: 'Icomoon';
  src: font-url("icomoon.eot");
  src: font-url("icomoon.eot?#iefix") format("embedded-opentype"),
       font-url("icomoon.svg#icomoon") format("svg"),
       font-url("icomoon.woff") format("woff"),
       font-url("icomoon.ttf") format("truetype");
  font-style: normal;
  font-variant: normal;
  font-weight: normal;
}

[class^="icon-"],
[class*=" icon-"] {
  font-family: "icomoon";
```

(https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face)

**Comment by dkhgh** [comment]
we might wanna move this stuff to our typography partial, not sure...
